uint8 id
float64 dislong
float64 dislat
float64 confidence
float64 type
